packages/block-editor/src/components/index.js

```diff
@@ -165,6 +165,7 @@ export { default as __experimentalInspectorPopoverHeader } from './inspector-pop
 
 export { default as BlockEditorProvider } from './provider';
 export { default as useSetting } from './use-setting';
+export { useBlockCommands } from './use-block-commands';
```

Comment:
Do we need the `Block` in the name of this hook. It's already part of block editor so maybe it can be just `useCommands` which would stand for (use all block editor commands)

Reply:
I think it's fine to be granular? Is there any downside?